src/components/UploaderPanel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from 'react';
+import { Label, ErrorMessage } from 'lbh-frontend-react';
+import styles from '../styles/UploaderPanel.module.scss';
+
+const classNameFromProps = (props: Props) => {
+  let className = `${styles.panel}`;
+  if (props.set) className += ` ${styles.setPanel}`;
+  if (props.error) className += ` ${styles.errorPanel}`;
+  return className;
+};
+
+const UploaderPanel: FunctionComponent<Props> = (props: Props) => (
```

Comment:
You don't need to type the props as it's included in the `FunctionComponent` type

```suggestion
const UploaderPanel: FunctionComponent<Props> = (props) => (
```